etl/steps/data/meadow/artificial_intelligence/2023-06-21/epoch.py

```diff
@@ -58,7 +58,6 @@ def run(dest_dir: str) -> None:
     #
     tb = Table(df, short_name=paths.short_name, underscore=True)
     ds_meadow = create_dataset(dest_dir, tables=[tb], default_metadata=snap.metadata)
-
```

Comment:
Maybe an accidental commit here? Think you can remove it with:

`git checkout origin/master etl/steps/data/meadow/artificial_intelligence/2023-06-21/epoch.py`

Not a big deal either way, I guess just better practice to remove